sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items-mlflow.ipynb

```diff
@@ -0,0 +1,1146 @@
+    "\n",
+    "best_child_run_id = mlflow_parent_run.data.tags[\"automl_best_child_run_id\"]\n",
+    "print(\"Found best child run id: \", best_child_run_id)\n",
+    "\n",
+    "best_run = mlflow_client.get_run(best_child_run_id)\n",
+    "\n",
+    "print(\"Best child run: \")\n",
+    "print(best_run)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get best model run's metrics\n",
+    "Access the results (such as Models, Artifacts, Metrics) of a previously completed AutoML Run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "pd.DataFrame(best_run.data.metrics, index=[0]).T"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the best model locally\n",
+    "Access the results (such as Models, Artifacts, Metrics) of a previously completed AutoML Run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create local folder\n",
+    "import os\n",
+    "\n",
+    "local_dir = \"./artifact_downloads\"\n",
+    "if not os.path.exists(local_dir):\n",
+    "    os.mkdir(local_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download run's artifacts/outputs\n",
+    "local_path = mlflow_client.download_artifacts(\n",
+    "    best_run.info.run_id, \"outputs\", local_dir\n",
+    ")\n",
+    "print(\"Artifacts downloaded in: {}\".format(local_path))\n",
+    "print(\"Artifacts: {}\".format(os.listdir(local_path)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "# Show the contents of the MLFlow model folder\n",
+    "os.listdir(\"./artifact_downloads/outputs/mlflow-model\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 6. Register best model and deploy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6.1 Create managed online endpoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import required libraries\n",
+    "from azure.ai.ml.entities import (\n",
+    "    ManagedOnlineEndpoint,\n",
+    "    ManagedOnlineDeployment,\n",
+    "    Model,\n",
+    "    Environment,\n",
+    "    CodeConfiguration,\n",
+    "    ProbeSettings,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Creating a unique endpoint name with current datetime to avoid conflicts\n",
+    "import datetime\n",
+    "\n",
+    "online_endpoint_name = \"ic-ml-fd-items-\" + datetime.datetime.now().strftime(\n",
```

Comment:
Minor: Use prefix "ic-ml-fridge-items-" all across endpoints, deployments and models to be consistent ?